specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerSecurityAdminConfiguration.json

```diff
@@ -870,75 +805,74 @@
         "provisioningState": {
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
-          "description": "The provisioning state of the security Configuration resource."
-        },
-        "appliesToGroups": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/NetworkManagerSecurityGroupItem"
-          },
-          "description": "Groups for configuration"
+          "description": "The provisioning state of the resource."
         }
       },
       "required": [
         "protocol",
         "access",
         "direction"
       ],
-      "description": "Security rule resource."
+      "description": "Security admin rule resource."
     },
-    "UserRule": {
+    "DefaultAdminRule": {
       "properties": {
         "properties": {
-          "$ref": "#/definitions/UserRulePropertiesFormat",
+          "$ref": "#/definitions/DefaultAdminPropertiesFormat",
           "x-ms-client-flatten": true,
-          "description": "Indicates the properties of the security user rule"
-        },
-        "systemData": {
-          "readOnly": true,
-          "description": "The system metadata related to this resource.",
-          "$ref": "./network.json#/definitions/SystemData"
+          "description": "Indicates the properties of the security admin rule"
         }
       },
       "allOf": [
         {
-          "$ref": "./network.json#/definitions/ProxyResource"
+          "$ref": "#/definitions/BaseRule"
         }
       ],
-      "description": "Network security admin rule."
+      "description": "Network default admin rule.",
+      "x-ms-discriminator-value": "DefaultAdminRule"
     },
-    "UserRulePropertiesFormat": {
+    "DefaultAdminPropertiesFormat": {
       "type": "object",
       "properties": {
         "displayName": {
           "type": "string",
+          "readOnly": true,
           "description": "A friendly name for the rule."
         },
         "description": {
           "type": "string",
+          "readOnly": true,
           "description": "A description for this rule. Restricted to 140 chars."
         },
+        "flag": {
```

Comment:
is this truly a free-form string and not an enum?

Reply:
After discussion, we would like it as string for now. And we will change it to enum in next version